lib/language_pack/helpers/node_installer.rb

```diff
@@ -18,7 +18,11 @@ def install
     # clean up un-used files. Instead we untar in a temp directory which
     # helps us avoid accidentally deleting code out of the user's slug by mistake.
     Dir.mktmpdir do |dir|
-      node_bin = "#{binary_path}/bin/node"
+      if ENV.include?('BUILDPACK_NODEJS_VENDOR_URL')
+        node_bin = "./bin/node"
+      else
+        node_bin = "#{binary_path}/bin/node"
+      end
```

Comment:
```suggestion
      node_bin = ENV.include?('BUILDPACK_NODEJS_VENDOR_URL') ? "./bin/node" : "#{binary_path}/bin/node"
```
in general i think rubyists prefer one line ternaries over if / else